tests/tests/son_operations_tests.cpp

```diff
@@ -732,7 +732,7 @@ BOOST_AUTO_TEST_CASE( son_heartbeat_test ) {
          PUSH_TX( db, trx, ~0);
          generate_block();
          trx.clear();
-         BOOST_CHECK( obj->status == son_status::in_maintenance);
+         BOOST_CHECK( obj->status == son_status::request_maintenance);
```

Comment:
same as one of the above heartbeat comments.

Reply:
ok, I'll fix it